geode-junit/src/main/java/org/apache/geode/security/ExpirableSecurityManager.java

```diff
@@ -31,44 +32,44 @@
  *
  * make sure to call reset after each test to clean things up.
  */
-public class ExpirableSecurityManager extends SimpleSecurityManager {
+public class ExpirableSecurityManager extends SimpleSecurityManager implements Serializable {
```

Comment:
This is `Serializable only` because of lambda style VM invocations, right?

Since this is in the product code and doesn't really need to go on the wire, you should instead subclass `ExpirableSecurityManager` as an inner-class within the dunit test and then use that subclass in the tests.
```
private static class SerializableExpirableSecurityManager 
    extends ExpirableSecurityManager
    implements Serializable {
...
}
```

Reply:
No, this is not production code. `ExpirableSecurityManager` is a test security manager in `geode-junit` module

Reply:
Oh yeah I see `geode-junit` now. Thanks!